vscode-typescript/track.yml

```diff
@@ -0,0 +1,50 @@
+slug: vscode-typescript
+id: 0dxngfkws4op
+type: track
+title: VSCode & TypeScript
+teaser: Edit and test TypeScript applications in VSCode
+description: |-
+  A basic track for editing a TypeScript application and running tests for it.
+  The track checks are failing based on the test results.
+
+  ## Using this template
+  To create your own track from this template, run this command:
+  ```
+  instruqt track create \
+      --from templates/container \
```

Comment:
I believe this should be `templates/vscode-typescript`

Reply:
Updated in https://github.com/instruqt/templates/pull/4.